packages/compass-connections-navigation/src/tree-data.ts

```diff
@@ -8,16 +8,35 @@ import type {
   VirtualPlaceholderItem,
   VirtualTreeItem,
 } from './virtual-list/use-virtual-navigation-tree';
+import { ConnectionStatus } from '@mongodb-js/compass-connections/provider';
 
-type Collection = {
-  _id: string;
+type Status = 'initial' | 'fetching' | 'refreshing' | 'ready' | 'error';
```

Comment:
we can now rename this to `DatabaseStatus`?

Reply:
Addressed this in [e9c23f2](https://github.com/mongodb-js/compass/pull/5886/commits/e9c23f28f4bea911a6464260e916cec7e9bf5107)